packages/cli/src/controller/init-controller.ts

```diff
@@ -16,8 +17,14 @@ export async function createProject(projectName: string): Promise<void> {
     currentPackage.name = projectName;
     const newPackage = JSON.stringify(currentPackage, null, 2);
     fs.writeFileSync(`${localPath}/package.json`, newPackage, 'utf8');
+    rimraf(`${localPath}/.git`, function (err) {
+      if (err) throw err;
+    });
```

Comment:
better to init a new git repo for the user after the one is deleted.

Reply:
That can be done in another PR.
I don't see a lot of cli tools have done that. It might exceed user's expectation.